doc/design.rst

```diff
@@ -580,21 +580,12 @@ The restic backup program guarantees the following:
 -  Accessing the unencrypted content of stored files and metadata should
    not be possible without a password for the repository. Everything
    except the metadata included for informational purposes in the key
-   files is encrypted and authenticated.
-
+   files is encrypted and authenticated. The cache is also encrypted by 
+   default to also prevent metadata leaks when encrypted filesystems are 
+   mounted.
 -  Modifications (intentional or unintentional) can be detected
    automatically on several layers:
 
```

Comment:
Is there a reason why the modification detection list was removed? The attacker examples look at tampered backups from a different angle.

Reply:
Yes, though the colon should have been replaced with a period. It was removed for a couple of reasons.
1. @fd0 had also suggested removing implementation specific details as much as possible so my thinking was explaining that modification can be detected was sufficient without including the extra details. See https://github.com/restic/restic/issues/208#issue-92063294. 
2. In the same comment, @fd0 mentioned the threat model "should be updated to the latest structure, e.g. there is no id or version file any more."  The detection list still referenced an `ID`. If I missed a detail of the structure and that's an incorrect interpretation, let me know.